src/scss/components/color/_index.scss

```diff
@@ -36,7 +36,7 @@
 	*/
 		// @debug output();
 
-		@if output(background-colors) {
+		@if output("background-colors") {
 			.bg-#{to-lower-case($color-name)},
```

Comment:
Als we de BEM methode willen eren zou ik het netter vinden als we `background--#{$color-name}` en `text--#{$color-name}` zouden gebruiken/toevoegen. :)

Reply:
Eensch.. alleen zo lang

Reply:
`bg--#{$color-name}` is ook al beter